src/main/mw.c

```diff
@@ -419,10 +420,23 @@ void processRx(void)
     }
 
     throttleStatus_e throttleStatus = calculateThrottleStatus(&masterConfig.rxConfig, masterConfig.flight3DConfig.deadband3d_throttle);
+    rollPitchStatus_e rollPitchStatus =  calculateRollPitchCenterStatus(&masterConfig.rxConfig);
 
+    /* In airmode Iterm should be prevented to grow when Low thottle and Roll + Pitch Centered. */
+    /* This is needed to prevent Iterm winding on the ground, but keep full stabilisation on 0 throttl in air */
```

Comment:
@borisbstyle : As I read the code :
- when `(antiWindupProtection || motorLimitReached) == false`, actual I term is sampled into `errorGyroILimit`
- when one condition becomes true, `errorGyroI` is constrained to +-`errorGyroILimit`

So when fly with stick centered on balanced plane, `errorGyroI` is almost zero. Turning throttle down will impose last `errorGyroI` as I term limit. So when I want straight glide, I get almost no I term correction, any disturbance will be corrected only with P+D term.

Implementation of `scaleItermToRcInput` will make it even worse - when I quickly turn throttle from 70+% to 0% - sampled `errorGyroI` will be almost zero.